### PR TITLE
Surface Google Autoscaler information in server group details.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

File: app/scripts/modules/google/serverGroup/details/scalingPolicy/scalingPolicy.directive.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<dl ng-class="InsightFilterStateModel.filtersExpanded ? '' : 'dl-horizontal dl-wide'" style="margin-bottom: 20px">
+  <dt>
+    Target {{policy.basis}}
+    <help-field ng-if="policy.basisHelpKey" key="{{policy.basisHelpKey}}"></help-field>
```
Comment (anotherchrisberry) [inline app/scripts/modules/google/serverGroup/details/scalingPolicy/scalingPolicy.directive.html]: You shouldn't need the `ng-if` here - if the `key` is not found, the help bubble will not render.

**Comment by anotherchrisberry** [comment]
LGTM once that `ng-if` is removed.

File: app/scripts/modules/google/serverGroup/details/scalingPolicy/scalingPolicy.directive.html (hunk 1 above)
Comment (duftler) [inline app/scripts/modules/google/serverGroup/details/scalingPolicy/scalingPolicy.directive.html]: Done.

**Comment by duftler** [comment]
image

**Pull request merged**